libraries/AP_TemperatureSensor/TSYS01.cpp

```diff
@@ -15,6 +15,9 @@ static const uint8_t TSYS01_CMD_READ_ADC    = 0x00;
 
 bool TSYS01::init()
 {
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    return true;
```

Comment:
This should probably be false right? We didn't actually init the sensor.

Reply:
I think it should be true as on SITL, we are considering that the sensor is healthy and report a temperature.
@jaxxzer, on your call !

Reply:
ping @jaxxzer

Reply:
I think that we should make it false; then this is a non-functional change (besides eliminating the warnings). We don't have support in ::_read_adc() for sitl.